### PR TITLE
member-list-component

### PR DESCRIPTION
**Pull request by AzizaAlSayed**
Convent Member list component to member component ( array mapping )

### CODE REVIEW

File: src/components/member.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { members } from '../data.js';
+
+export default function Member() {
```
Comment (Abdallatif) [inline src/components/member.js]: Make the `Member` component only responsible for rendering 1 member. And add a new component `MemberList` that takes `members` as a prop and map it to `Member` in jsx

File: src/components/member.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    const membersList = members.map(member =>
```
Comment (Abdallatif) [inline src/components/member.js]: `members` should be a prop

File: src/components/App.js [hunk 3]
```diff
@@ -1,8 +1,7 @@
 import Header from './header';
 import Search from './search.js';
```
Comment (Abdallatif) [inline src/components/App.js]: you don't have to specify `.js` in import

**Review by Abdallatif — CHANGES_REQUESTED** [review]

File: src/components/member.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        <div className="FlexDisplay">
```
Comment (Abdallatif) [inline src/components/member.js]: you should pass a `key` for every element in map

File: src/components/App.js [hunk 5]
```diff
@@ -1,30 +1,22 @@
 import Header from './header';
-import Search from './search.js';
-import MemberList from './memberList.js';
-import NewMember from './newMember.js';
-
+import Search from './search';
+import Member from './member';
+import NewMember from './newMember';
 import '../App.css';
+import { members } from '../data.js';
```
Comment (Abdallatif) [inline src/components/App.js]: you don't need `.js` in import

File: src/components/memberList.js [hunk 6]
```diff
@@ -1,27 +1,10 @@
-export default function MemberList({ name }) {
+import Member from "./member";
+
+export default function MemberList({ members }) {
+  const membersList = members.map(member =>
```
Comment (Abdallatif) [inline src/components/memberList.js:4]: not important but you can return the map in the jsx
```jsx
return (
  <p>
    {members.map(member => <Member id={member.id} name={member.name} />}
  </p>
);
```

File: src/components/memberList.js [hunk 7]
```diff
@@ -1,27 +1,10 @@
...
+    <Member id={member.id} name={member.name} ></Member>
```
Comment (Abdallatif) [inline src/components/memberList.js]: don't forget to pass `key`

**Review by Abdallatif — CHANGES_REQUESTED** [review]

File: src/components/member.js (hunk 4 above)
Comment (AzizaAlSayed) [reply src/components/member.js]: for all tags or for all members?

Comment (Abdallatif) [reply src/components/member.js]: No just the element in the top level. Consider this resolved.

**Review by Abdallatif — APPROVED** [review]

**Pull request merged**